package-lock.json

```diff
@@ -0,0 +1,2873 @@
+{
```

Comment:
I think this file is also generated, so we may want it in .gitignore instead of committing it. But I'm not 100% sure about that.

Reply:
the instructions said to only ignore a folder called node_modules/ so I'm not exactly sure myself.

Reply:
According to stackoverflow (which should always be treated as gospel 😄 ), this should indeed be committed:
https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5

So you're right!